bitcoin-protocol-development/p2p.md

```diff
@@ -7,12 +7,12 @@ _Note that this section depends on reviewing the network propagation resources o
 | Content | Time \(min\) |
 | :--- | :--- |
 | [Map of the Bitcoin Network](https://medium.com/@gloriazhao/map-of-the-bitcoin-network-c6f2619a76f3) | 20 |
-| Attacking p2p of Bitcoin Core w/ Amiti Uttarwar \([video \(until min 41\)](https://youtu.be/H-wH6mY9pZo?t=257) or [transcript](https://btctranscripts.com/la-bitdevs/2020-04-16-amiti-uttarwar-attacking-bitcoin-core/)\) | 35 |
```

Comment:
This is intentionally different. This video is the better version of the talk and is not the same as the one in the transcript.